tests/test_io.py

```diff
@@ -263,6 +263,38 @@ def test_write_filterSOS(filterSOS, tmpdir):
     assert actual == filterSOS
 
 
+def test_write_fractionaloctave_bands(tmpdir):
+    """ dsp.filter.FractionalOctaveBands
+    Make sure `read` understands the bits written by `write`
+    """
+    filename = os.path.join(tmpdir, 'fractionaloctave_bands.far')
+    fractionaloctave_bands = pyfar.dsp.filter.FractionalOctaveBands(
+        frequency_range=(20, 10000))
```

Comment:
It would be good to save the filter with an non-empty state to see if everything is written and read. Could for example be done by
```python
fractionaloctave_bands.init_state((1, ))
fractionaloctave_bands.process(pf.signals.sine(1000, 441), True)
```

Would also be good for the reconstructing filter bank